### ISSUE TITLE
License

### ISSUE THREAD

**Issue by swissspidy** [berlindb/core#116]
I just saw that BerlinDB is under GPL. Would you be open to possibly relicense as MIT or something or dual-license (like Gutenberg is planning)? This would allow using it in non-GPL plugins (which are allowed in the repo)

**Comment by JJJ** [comment]
Definitely open to whatever is most open. 👐

**Comment by swissspidy** [comment]
IANAL, but since any chosen license would need to be compatible with GPL (so it can be used in GPL-licensed plugins), the MIT license seems to be a good fit. It is also compatible with the Apache license, so it could be used in an Apache-licensed plugin as well. 

Of course existing contributors would need to agree to any change.

**Comment by JJJ** [comment]
I agree @swissspidy. MIT it is then! 🚀

Will need a YES👍 or NO👎 from these contributors:

- [ ] @JJJ 
- [ ] @szepeviktor
- [ ] @pippinsplugins
- [ ] @alexstandiford
- [ ] @ashleyfae

**Comment by szepeviktor** [comment]
- [x] @szepeviktor

**Comment by ashleyfae** [comment]
All set. :smile:

**Comment by JJJ** [comment]
PR ready for scrutiny.

I added myself as the lone copyright callout for 2 reasons:
* I wrote the bulk of what's here
* I can really only approve of adding myself

It doesn't seem like a good idea to maintain a list of every contributor in the LICENSE file, but it does seem like a good idea to list somebody in there for the sake of accountability – which also seems common in other MIT License usages I looked into.

Open to all feedback. 🍕
